src/v2i-hub/TelematicBridgePlugin/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "TelematicBridge",
+    "description": "Plugin that listens for TMX messages and forward them to the Telematic cloud services.",
+    "version": "@PROJECT_VERSION@",
+    "exeLocation": "/bin/TelematicBridgePlugin",
+    "coreIpAddr": "127.0.0.1",
+    "corePort": 24601,
+    "messageTypes": [],
```

Comment:
Update Message Types here if we want status counts for the messages this plugin handles.